.github/workflows/release.yml

```diff
@@ -0,0 +1,127 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      module:
+        description: "Maven (Sub)module to Release (e.g.,
+                  keycloak-extensions,
+                  authenticator-required-action,
+                  keycloak-extension-rest,
+                  user-management).
+                  Parent is the default"
+        required: false
+        default: "parent"
+      releaseVersion:
+        description: "Version to use for release."
+        required: true
+        default: "X.Y.Z"
+      developmentVersion:
+        description: "Version to use for next development cycle."
+        required: true
+        default: "X.Y.Z-SNAPSHOT"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Configure Git User
+        run: |
+          git config user.email "actions@github.com"
+          git config user.name "GitHub Actions"
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: 17
+          cache: 'maven'
+
+      - name: Install Parent POM (If necessary locally)
+        run: mvn install -N || echo "Parent-POM already exists or is getting downloaded from repo."
```

Comment:
This will always execute maven and install the parent module to the local maven repository.
Therefore, I think the `echo` statement is somewhat missleading. what is the purpose of that?